js/targetCowFunctions.js

```diff
@@ -0,0 +1,85 @@
+$('body').on('click', '.js_clickable_cow', function(e) {
+    save_cow(e)
+})
+
+/**
+ * when the cow is clicked, remove the class js_clickable_cow, add class rocket_cow and execute rocket cow function
+ */
+function save_cow(e) {
+    var clicked_cow = e.target
+    var $cow_container = $(clicked_cow).parent()
+    $(clicked_cow).removeClass('js_clickable_cow')
+    $cow_container.addClass('rocket_cow')
+    rocket_cow($cow_container)
+}
+
+/**
+ *increases score on screen
+ */
+function increment_score() {
+    var score_element = parseInt($(".score_value").text()) + 1
+    $(".score_value").text(score_element)
+}
+
+/**
+ *changes value of lives on screen, when lives is equal to zero end game
+ */
+function lose_life() {
+    var lives_element = get_lives() - 1
```

Comment:
lives_element is incorrect, its not the element, its the amount of remaining lives